content/spin/spin-oci.md

```diff
@@ -141,6 +161,16 @@ Lastly, let's run this Spin application.
 
 {{ blockEnd }}
 
+{{ startTab "v0.10.0"}}
+
+<!-- @selectiveCpy -->
+
+ ```bash
+ spin up --from-registry ghcr.io/USERNAME/spin-react-fullstack:v1
```

Comment:
Consider using `-f` here.  You should only need to use `--from-registry` if Spin can't recognise the string as a registry reference.  (Please tell me if this isn't working for your test case!)

Reply:
I have the `-f` for the `0.10.0` version (just below that `0.9.0` tab syntax).
Wes `-f` `--from` implemented in `0.9.0`? If so happy to update as per this review commend; otherwise the `0.10.0` tab is useful to show the difference between running `.9` and `.10` just in case a user is on an older version and following these docs.

Reply:
![Screenshot 2023-03-06 at 9 47 28 am](https://user-images.githubusercontent.com/9831342/222993161-6d8067c6-0e24-479e-a08a-d3e74b926465.png)

Reply:
I'd potentially put v0.10.0 before v0.9.0 ad also if there is no change we should possibly just make the blocks `>=v0.9.0`, just a thought.

Reply:
Sure, that's a great point @karthik2804 
Let me re-org and ping back here when ready.
Gimme a sec ...

Reply:
@tpmccallum No, you are correct, `-f` is 0.10 onwards.  Apologies, I didn't open the surrounding lines and misread this as new 0.10 content - sorry!  _disappears into hedge_

Reply:
@karthik2804 I quite like the explicitness of spelling out individual versions, and I think it plays nicely with sticky tabs?  So if I select 0.9 then I get 0.9 _everywhere_ rather than needing to select `>= 0.9` / `<= 0.9` / `0.8 though 0.11` etc. as I move through different parts of the docs.

Reply:
+1 to latest version first though

Reply:
Ok, that looks great; we should make that a convention i.e. the latest version from left to right in the multi-version code block.
![Screenshot 2023-03-06 at 10 00 39 am](https://user-images.githubusercontent.com/9831342/222993767-c7bc0c16-a515-4053-a7b5-83fdc5840775.png)

Reply:
Also, having more than two versions (like before `0.8.0`, `0.9.0`, & `0.10.0`) was probably unnecessary and too busy for the eye. Great feedback on the `>=`

Reply:
Thanks @itowlson 
This is a good feedback hybrid; let me revise it slightly, and I think we might have it nailed. Another sec ...

Reply:
Done